aioredis/commands/sorted_set.py

```diff
@@ -18,7 +18,10 @@ class SortedSetCommandsMixin:
     ZSET_AGGREGATE_MIN = 'ZSET_AGGREGATE_MIN'
     ZSET_AGGREGATE_MAX = 'ZSET_AGGREGATE_MAX'
 
-    def zadd(self, key, score, member, *pairs):
+    SET_IF_NOT_EXIST = 'SET_IF_NOT_EXIST'  # NX
+    SET_IF_EXIST = 'SET_IF_EXIST'  # XX
```

Comment:
Ouch, these two constants will conflict with the same ones in `aioredis/commands/string.py`.
Please rename both to `ZSET_IF_NOT_EXIST / ZSET_IF_EXIST`